appveyor.yml

```diff
@@ -1,10 +1,19 @@
-version: 4.0.0.{build}
+version: 4.1.0.{build}
 platform:
-    - x64
+  - x64
```

Comment:
http://www.appveyor.com/docs/appveyor-yml, when I was thinking about testing 32-bit and 64-bit Windows. Didn't end up using it.